tests/messages/test_message.py

```diff
@@ -137,3 +137,76 @@ def test_load(self):
         )
 
         Message.message_type = None
+
+    def test_load_message_id_unvalid(self):
+        payload = (
+            '{"message_id": "foo", '
+            '"message_type": "scan.start", '
+            '"group_id": "866350e8-1492-497e-b12b-c079287d51dd", '
+            '"created": 1628512774.0}'
+        )
+        Message.message_type = MessageType.SCAN_START
+        self.assertRaises(ValueError, Message.load, payload)
```

Comment:
IMHO using the with statement for assertRaises is much more readable
```suggestion
        with self.assertRaises(ValueError):
            Message.load(payload)
```